packages/cozy-intent/src/view/hooks/useIsAvailable.ts

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from 'react'
+
+import { useWebviewIntent } from './useWebviewIntent'
+
+export const useIsAvailable = (methodName: string): boolean => {
+  const webviewIntent = useWebviewIntent()
+  const [isAvailable, setIsAvailable] = useState(false)
```

Comment:
Do you thinl we should return undefined or null when the webviewIntent.call has not been made ?

Reply:
si on veut tester ce cas, un simple `!isAvailable` ne suffira pas il faudra être explicite avec un `isAvailable === something`. Et c'est plus court je pense de tester avec `null` que `undefined` :p

Reply:
Ça retourne null maintenant par défaut : 
- permet de tester si on veut ce cas
- toujours possible de faire un simple if(isAvailable)